lemon/market_data/live_streaming.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+from typing import List, Optional
+
+from lemon.base import Client
+from lemon.market_data.model import Token
+from lemon.types import Sorting
+
+
+class LiveStreaming:
+    def __init__(self, client: Client):
+        self._client = client
+
+    def get_token(self) -> Token:
```

Comment:
`get_token` -> `auth`

Reply:
You sure?

Reply:
let's keep the names of the endpoints similar to their url names (we may want to generate the API from swagger in the future - this will help) ... you can call it auth if you want.